website/content/api-docs/system/replication/replication-performance.mdx

```diff
@@ -540,8 +540,22 @@ $ curl \
 
 ## Update performance secondary's primary
 
-This endpoint changes a performance secondary cluster's assigned primary cluster using a
-secondary activation token. This does not wipe all data in the cluster.
+This endpoint changes a performance secondary cluster's assigned primary cluster. This does not
+wipe all data in the cluster.
+
+There are two different ways to use this API call, each with its own parameter. The first way
+is using a secondary activation token (`token` in the parameter list below). This is the original
+way to call `update-primary`. When this is used, Vault does a lot of extra work behind the scenes.
+
+The other way is with the `update_primary_addrs` parameter, which was added in Vault 1.13. This is an array
+of Vault cluster addresses that represent the cluster listeners of the primary cluster. These addresses will
+be pinged one at a time via gRPC and the ones that respond successfully will be used as the secondary's new
+primary addresses. This is a quicker process vs using a secondary activation token. Vault does
+less work behind the scenes, which means potentially less disruption for replication. This method
+works best for performance secondary clusters.
+
+It's important to note that `update_primary_addrs` and `token` are mutually exclusive. You may use
+one or the other, but not both.
```

Comment:
```suggestion
There are two ways to update the primary assignment:
	
1. **Use a secondary activation token with the `token` parameter**.
1. **Use primary cluster addresses with the `update_primary_addrs` parameter**.
   During the update, cluster addresses are pinged one at a time via gRPC. The
   first cluster to respond successfully is assigned as the new primary address.

The two update methods are mutually exclusive. You may use one or the other,
but not both. We recommend updating primary cluster assignments with cluster
addresses whenever possible. Address updates are much faster than activation
token updates because Vault does less work behind the scenes, which means
potentially less disruption in replication.
```

Style correction: write in active voice, avoid "this" as a pronoun, make key information easy to scan, focus on current functionality